internal/02_sharing_demo.md

```diff
@@ -24,27 +25,26 @@ drive_upload("demo.txt", "Happy Little Demo")
 
     ## File uploaded to Google Drive: 
     ## demo.txt 
-    ## As the Google document named:
+    ## As the Google text/plain named:
```

Comment:
Sidebar: once we have human-friendly nicknames stored for the mime types, this will be a great place to use them.